vendor/bat-native-ads/src/bat/ads/internal/bundle/bundle.cc

```diff
@@ -257,6 +361,16 @@ void Bundle::OnCreativeAdNotificationsSaved(
   BLOG(3, "Successfully saved creative ad notifications state");
 }
 
+void Bundle::OnCreativeNewTabPageAdsSaved(
+    const Result result) {
+  if (result != SUCCESS) {
+    BLOG(0, "Failed to save creative new tab page ads state");
```

Comment:
Might be out-of-scope, but is it possible to add retry if it's not too hard and keep a counter on the number of retries?

Reply:
Discussed and we retry the next time we download a catalog